charts/jenkins/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: jenkins
 home: https://jenkins.io/
-version: 3.8.8
+version: 3.8.9
```

```suggestion
version: 3.9.0
```

Let's bump minor version as it's kind of a new feature.